csv-validator-application/src/assembly/filter.properties

```diff
@@ -0,0 +1,10 @@
+#
```

Comment:
Should this be renamed to `filter-nix-with-jre.properties` to be consistent with the nameing of the Windows specific file?

Reply:
>  It will be "without jre"
@sparkhi I am a bit confused by this! There are already versions without a JRE available in the form of the published artefacts from the assemblies for csv-validator-ui and csv-validator-cmd. So why add another that is effectively the same?

Also why produce a JRE version only for Windows? What about macOS and Linux users?